docs/architecture/security/principles/01-servers-are-not-trusted.mdx

```diff
@@ -0,0 +1,37 @@
+# P01 - Servers are not trusted
+
+Clients do not need to trust Bitwarden server or web infrastructure to be good actors in supplying
```

Comment:
I do not think this goes far enough. I'm not sure how to word it, but I want to include not just the actual server, but anything that from the client's perspective is the server (i.e include malicious MITMs that *do not have database access).

Reply:
That's what I was going for with "or web infrastructure," but perhaps being second implies _bitwarden owned_ web infrastructure? that's not the intent and I'm open to changing to clarify.